skype/skype-ps/skype/Get-CsOnlineUser.md

```diff
@@ -206,30 +269,14 @@ In Example 2 information is returned for a single online user: the user with the
 
 ### -------------------------- Example 3 --------------------------
 ```
-Get-CsOnlineUser -Filter {ArchivingPolicy -eq "RedmondArchiving"}
+Get-CsOnlineUser -Filter "AccountEnabled eq "Disabled""
 ```
 
-Example 3 uses the Filter parameter to limit the returned data to online users who have been assigned the per-user archiving policy RedmondArchiving.
-To do this, the filter value {ArchivingPolicy -eq "RedmondArchiving"} is employed; that syntax limits returned data to users where the ArchivingPolicy property is equal to (-eq) "RedmondArchiving".
+Example 3 uses the Filter parameter to limit the returned data for users who have their accounts disabled.
 
-### -------------------------- Example 4 --------------------------
-```
-Get-CsOnlineUser -Filter {HideFromAddressLists -eq $True}
-```
-
-Example 4 returns information only for user accounts that have been configured so that the account does not appear in Microsoft Exchange address lists.
-(That is, the Active Directory attribute msExchHideFromAddressLists is True.) To carry out this task, the Filter parameter is included along with the filter value {HideFromAddressLists -eq $True}.
 
-### -------------------------- Example 5 --------------------------
-```
-Get-CsOnlineUser -Filter {TenantId -eq "bf19b7db-6960-41e5-a139-2aa373474354"}
-```
 
-The command shown in Example 5 returns information for all the online users assigned to the tenant with the TenantID "bf19b7db-6960-41e5-a139-2aa373474354".
-To accomplish the task, the command includes the Filter parameter along with the filter value {TenantId -eq "bf19b7db-6960-41e5-a139-2aa373474354"}.
-This filter limits the returned data to online users assigned to the tenant "bf19b7db-6960-41e5-a139-2aa373474354".
-
-### -------------------------- Example 6 --------------------------
+### -------------------------- Example 4 --------------------------
 ```
 PS C:\> $MeetingPolicy="Kiosk"
 PS C:\> $filterString = 'TeamsMeetingPolicy -eq "{0}"' -f $MeetingPolicy
```

Comment:
based on the changes in operators mentioned on this commit, is this example still valid?

Reply:
@get-itips thanks for pointing out, yes I need to change that